documentation/continuous-integration.md

```diff
@@ -0,0 +1,34 @@
+<!-- @license CC0-1.0 -->
+
+# Continuous integration
+
+## Commit hashes for GitHub Actions
+
+We use commit hashes instead of version numbers for GitHub Actions in our workflows.
```

Comment:
This clarifies what we use them for.

```suggestion
We use commit hashes instead of version numbers to select which releases of GitHub Actions we use in our workflows.
```

Reply:
Yes, much better. I had been fiddling around a bit already but couldn’t find the way out.